opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/authenticate/StartConsentAuthorization.java

```diff
@@ -87,6 +90,28 @@ protected void doRealExecution(DelegateExecution execution, Xs2aContext context)
         execution.setVariable(CONTEXT, context);
     }
 
+    private Response<StartScaprocessResponse> getScaStart(Xs2aContext context, ValidatedPathHeaders<Xs2aInitialConsentParameters, Xs2aStandardHeaders> params) {
```

Comment:
Please synchronize this logic with @clovisgakam. Targo bank has very similar handling, agree on some single apporoach